src/Microsoft.Azure.WebJobs.Extensions.Sql.csproj

```diff
@@ -48,7 +48,7 @@
     </Reference>
     <None Include="pkgicon.png" Pack="true" PackagePath="" />
     <None Include="..\lib\Microsoft.SqlServer.TransactSql.ScriptDom.dll" Pack="true" PackagePath="lib/$(TargetFramework)" />
-    <None Include=".\README.md" Pack="true" PackagePath="" />
+    <None Include="..\README.md" Pack="true" PackagePath="" />
```

Comment:
Oh hey good catch, yeah let me update them to be fully qualified

Reply:
It's almost like I have experience with markdown links or something 😅

Reply:
It's not quite as nice since now : 

- It always links to main (so if you have a specific branch with changes you wanted to test or whatever the links would go to the wrong place)
- If editing in VS Code it'll open up in your external web browser instead of in VS Code itself

Neither of these are huge issues - just minor things during development. So I'm fine with making this since the alternative is to keep two copies of the README (which makes keeping them in sync annoying) or write some custom script to replace the relative values before packaging. Both of which seem like more work than is necessary.